app/models/request.rb

```diff
@@ -4,6 +4,7 @@ class Request < ApplicationRecord
   include OwnerField
 
   acts_as_tenant(:tenant)
+  acts_as_tree
```

Comment:
This call does not require any configuration?

Reply:
The optional parameter is `:sort` and we already sort requests by creation time.

Reply:
What if that current sort goes away?  The API is closer to supporting a `sort_by` option and we would likely remove the hardcoded value from the model and allow the UI to request the sort order.  The current default scope seems to supersedes any other  sort value passed so it limits what the API callers (like the UI) can do.

Does it make sense to define it if the default scope was not there?

Reply:
sorting is optional for `ActsAsTree`. If we depend on `sort_by` option from caller, we have more reason not  to add the `:sort` option in `acts_as_tree()`.